recipes/fs-gcsfs/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fs-gcsfs" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "12380577bb0e5f275e49c1c734c4ffa25f59cb720243255cee96899994b9cd71" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python ==3.*, >=3.5
+    - google-cloud-storage ==1.*, >=1.10.0
+    - fs ==2.*, >=2.0.0
+
+test:
+  imports:
+    - fs_gcsfs
+
+about:
+  home: https://github.com/Othoz/gcsfs
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: A Python filesystem abstraction of Google Cloud Storage (GCS) implemented as a PyFilesystem2 extension
+  doc_url: https://fs-gcsfs.readthedocs.io/
+  dev_url: https://github.com/Othoz/gcsfs
+
+extra:
+  recipe-maintainers:
+    - birnbaum
+    - mathiaseitz
```

Comment:
@mathiaseitz, @trendelkampschroer, are you ok being added as maintainers?

Reply:
Yes, that's perfect. Thank you.

Reply:
Yes. Thanks!